script/npm_test_plugin.js

```diff
@@ -20,7 +20,8 @@ var { allPluginDirs, skippedDirsKeys, dirsKeys } = filterPluginDirectories();
 const passedArgs = process.argv.slice(2);
 const coreConfigPath = path.resolve(__dirname, '../webpack/jest.config.js');
 const coreConfig = require(coreConfigPath);
-process.env.NODE_OPTIONS = '--max-old-space-size=8192 ' + (process.env.NODE_OPTIONS || '');
+process.env.NODE_OPTIONS = `--max-old-space-size=8192 ${process.env
+  .NODE_OPTIONS || ''}`;
```

Comment:
The line break *before* the dot throws me off, but that's probably just because I am not used to JS notation?

Reply:
linebreaks are always before dots?
```js
function input_group_hidden(btn) {
  target = $(btn)
    .closest('.input-group')
    .find('textarea, input');
```

Reply:
Not in Python/Ruby :)